generators/server/templates/pom.xml.ejs

```diff
@@ -760,7 +760,7 @@
             <scope>test</scope>
         </dependency>
         <%_ } _%>
-        
```

Comment:
can remove this empty line

Reply:
Then we should also remove the other two empty lines @ L769 and L775, right?